lib/mongoose-store.js

```diff
@@ -45,13 +57,20 @@ Store.prototype.findById = function(id) {
 };
 
 Store.prototype.read = function(_id) {
-  return this.model.findOne({id: _id}).exec().then(convertToJSON);
+  return this.model.findOne({id: _id}).exec().then(function(foundDocument) {
```

Comment:
We missing catch here. I think that nothing will be returned if other error happened (lost database connection, backend overloaded etc.)? Best to supply catch handler as well.